Sources/PointFree/Stripe.swift

```diff
@@ -250,6 +253,28 @@ public struct Stripe {
 
     public typealias Id = Tagged<Subscription, String>
 
+    public struct Discount: Codable {
+      public private(set) var coupon: Coupon
+
+      public struct Coupon: Codable {
+        public typealias Id = Tagged<Coupon, String>
+
+        public private(set) var amountOff: Int?
+        public private(set) var id: Id
+        public private(set) var name: String
+        public private(set) var percentOff: Float?
+        public private(set) var valid: Bool
```

Comment:
these 5 fields are the minimal fields I think we need. we dont even need `amountOff` right now, but thought I would keep it around just in case.